sympy/sets/ordinals.py

```diff
@@ -1,36 +1,67 @@
 from sympy.core import Basic
 import operator
 
+class OrdinalPow(Basic):
```

Comment:
I would have expected that `OrdinalPow` would be used for general exponentiation of ordinals like `a^b` where `a` and `b` are both (arbitrary) ordinals. That is why I suggested the name `OmegaPower` for the special building blocks of Cantor normal form where the base is always `omega` and an extra multiplier, not present in general exponentiation, is appended.

Reply:
sorry , but I could not get it. 
> I would have defined a class (maybe OmegaPower) whose arguments are exponent and multiplicity (or, briefly, exp and mult), and then defined comparisons for them. (The arguments of Ordinal would then be objects of that class.)
>
I tried to do this . can you please give an example, so that I could implement it in a proper way

Reply:
The implementation looks good to me. I only think that the name of the class is not well chosen. (OrdinalPow could be saved to denote the actual exponentiation of ordinals, if necessary. It could then have `base` and `exp` that would both be ordinals.)